src/HotChocolate/Language/src/Language.Rewriters/DefaultSyntaxNavigator.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Collections.Generic;
+using static HotChocolate.Language.Rewriters.LangRewritersResources;
+
+namespace HotChocolate.Language.Rewriters;
+
+public class DefaultSyntaxNavigator : ISyntaxNavigator
```

Comment:
/// <summary>
/// Represents the default implementation of <see cref="ISyntaxNavigator" />
/// </summary>